packages/block-editor/src/components/inner-blocks/warning-max-depth-exceeded.native.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import { TouchableWithoutFeedback, View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+import { useDispatch, useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import Warning from '../warning';
+import BlockFallbackWebVersion from '../block-fallback-web-version';
+import { store as blockEditorStore } from '../../store';
+
+const WarningMaxDepthExceeded = ( { clientId } ) => {
+	const [ showDetails, setShowDetails ] = useState( false );
+
+	const { isSelected, innerBlocks } = useSelect(
+		( select ) => {
+			const { getBlock, isBlockSelected } = select( blockEditorStore );
+			return {
+				innerBlocks: getBlock( clientId )?.innerBlocks || [],
+				isSelected: isBlockSelected( clientId ),
+			};
+		},
+		[ clientId ]
+	);
+	const { replaceBlocks } = useDispatch( blockEditorStore );
+
+	const onUngroup = () => {
+		if ( ! innerBlocks.length ) {
+			return;
+		}
+
+		replaceBlocks( clientId, innerBlocks );
+	};
+
+	/* translators: Warning related to having blocks deeply nested. %d: The deepest nesting level. */
+	const descriptionFormat = __(
+		'Blocks with inner blocks that are nested deeper than %d levels might not be rendered properly in the native version of the editor. For this reason, we recommend flattening the content by ungrouping the block or editing the block using the web editor.'
+	);
+	const description = sprintf( descriptionFormat, MAX_NESTING_DEPTH );
+
+	return (
+		<TouchableWithoutFeedback
+			disabled={ ! isSelected }
+			accessibilityLabel={ __( 'Warning message' ) }
+			accessibilityRole={ 'button' }
+			accessibilityHint={ __( 'Tap here to show more details.' ) }
```

Comment:
I'm unsure as to why, but this button is announced as a "help" button. 

iOS: `Help button. Tap here to show help.`
Android: `Help button button. Tap here to show help. [Disabled|Double-tap to activate].` 

I believe the duplicative "button" term for Android is a preexisting issue. Do you know where the "help" term is originates?

Reply:
From searching, the a11y description match the Unsupported Block Interface. However, I do not understand why they would be presented here. 

https://github.com/WordPress/gutenberg/blob/e028e9ada1c06cb8e31e9b848e492ad37422ba65/packages/block-library/src/missing/edit.native.js#L86-L99

Reply:
Yeah, that's unexpected. Have you experienced this when testing the installable build? If so, this is likely the same issue as the one mentioned [here](https://github.com/WordPress/gutenberg/pull/54382#discussion_r1327312563). I'll update the installable builds to reflect all changes, sorry for the inconvenience 🙇 .